wiki/Beatmap/Pattern/osu!mania/Anchor/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Anchor
+
+**Anchors**, usually referring to a byproduct of [streams](/wiki/Beatmap/Pattern/osu!mania/Stream) or sometimes [chordjacks](/wiki/Beatmap/Pattern/osu!mania/Jack#chordjack), are a high concentration of notes in one column that appear consecutively within the same snap interval, typically in 1/2nd snap intervals. The name **anchor** comes from the motion the pattern creates, in which your finger is essentially anchored to that key due to the consecutive, intervallic presses. 
+
+![](img/anchorstream.png "This highlighted notes show an anchor within jumpstreams.")
+
+![](img/anchorjack.png "This highlighted notes show an anchor within chordjacks, they may also be referred to as longjacks in this context.")
```

Comment:
Oops my bad, I forgot to check this for awhile... anyways...

Firstly, the terms anchorstream and anchorjacks don't exist and fundamentally wouldn't make sense to use here since anchorstream implies a pattern that technically could never exist. I only named the images as such to keep the file name intact with the section (I suppose it should be anchor_stream.png + anchor_jack,png ?). Though the key difference being that the terms are _similar_ but not the _same_. 
Both images do not contain longjacks because an anchor is a blanket term whilst a longjack is not and requires a specific motion to be taken place (the jackhammer motion). For example, longjacks do not exist in jumpstream because you would not play them with a jackhammer motion. The terms only start to overlap at very high density of jumpstream because the strain and endurance aspects start to overshadow moreso, but it's still fundamentally a different motion.

Not sure if it would be worth mentioning that anchor is supposed to be a blanket term while longjacks are specific to jacks? I thought it was already implied from longjacks only being used for jack patterning, which would make sense (in my head atleast) because it implies that a jack is required. What do you think?

Reply:
do `anchor-within-jumpstreams.png` and `anchor-within-chordjacks.png` (note that image file names 
should use hyphens) sound better? i've edited the suggestion

i still don't think it's relevant to mention longjacks here. the article is about anchors and longjacks are explained elsewhere (a jack with 3+ notes)